src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -21,17 +21,31 @@ export const useStyles = makeStyles<DescriptionListProps>(
         display: 'grid',
         gap: theme.spacing(1),
         gridTemplateColumns: '1fr 3fr',
+        margin: 0,
         maxHeight: theme.pxToRem(432),
         minWidth: theme.pxToRem(224),
         overflowY: 'auto',
         listStyle: 'none',
-        paddingBottom: theme.spacing(1),
-        paddingLeft: 0,
-        paddingTop: theme.spacing(1),
+        padding: 0,
       },
       title: {
         color: theme.palette.text.primary,
         gridColumn: '1/-1',
+        marginBottom: theme.spacing(0.5),
+      },
+      hasTitleIcon: {
+        alignItems: 'center',
+        display: 'grid',
+        gap: theme.spacing(1),
+        gridTemplateColumns: `${theme.pxToRem(22)} 1fr`,
+        '& ~ li': {
+          marginLeft: theme.pxToRem(30),
+        },
```

Comment:
I would like some help refining here so I don't have to use this `~` sibling selector. 

I would prefer to leverage `!!TitleIcon` within `DescriptionTerm.tsx` and `DescriptionListGroupHeading.tsx` but I couldn't figure out how to do that.